nginx.conf

```diff
@@ -44,10 +44,10 @@ http {
         rewrite ^/(.*)//+(.*)   $scheme://$http_host/$1/$2  permanent; # remove adjacent slashes in the middle of the URI
         rewrite ^/([^.]*[^/])$  $scheme://$http_host/$1/    permanent; # force a trailing slash except for files with an extension
 
-        # Permanent redirects (301)
-        rewrite ^/docs/hostedservice/(.*)$                              $scheme://$http_host/docs/en/hosted-service/$1      permanent;
```

Comment:
Those `/docs/hostedservice/*` URLs must be really old; I couldn't find any reference to those anywhere. Also, we deprecated the HS, so I think it's safe to remove this rule.